### PR TITLE
Update miembros.md

### PR DESCRIPTION
**Pull request by Javier1995**
This can be better honestly

### CODE REVIEW

File: miembros.md [hunk 1]
```diff
@@ -21,4 +21,3 @@
 19. Talia al Ghul
 20. Lady Shiva
 21. Giganta
-22. Deadshot
```
Comment (Javier1995) [inline miembros.md:24]: Can you please add more villiants

Comment (pedromayo2020) [reply miembros.md:24]: Yes you can add more

**Review by pedromayo2020 — CHANGES_REQUESTED** [review]

File: miembros.md (hunk 1 above)
Comment (Javier1995) [reply miembros.md:24]: alrigh I will

**Review by pedromayo2020 — APPROVED** [review]

**Comment by Javier1995** [comment]
See you

**Comment by Javier1995** [comment]
cool

**Comment by pedromayo2020** [comment]
fine

**Pull request merged**